### PR TITLE
Notifications weren't working

### PR DESCRIPTION
**Pull request by mhuggins7278**
Messages were being sent to the room but the "notification" in the HC room wasn't happening.
The API doesn't like the url params it needs to be in the json object

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -47,12 +71,14 @@ module.exports= function(onConsoleLog,
   onConsoleError = onConsoleError || function(logEvent) {
     defaultHeaderString(logEvent);
     console.error.apply(null, logEvent.arguments);
+    notifyHC(logEvent.arguments);
```
Comment (wballard) [inline server.js]: I think this needs an env var 'if' guard of some kind

File: server.js [hunk 2]
```diff
@@ -62,12 +88,13 @@ module.exports= function(onConsoleLog,
     'exception': onException
   };
 
-  var process = function(request, flungLogs) {
+  var processLog = function(request, flungLogs) {
```
Comment (wballard) [inline server.js]: is this just a rename?

File: server.js (hunk 1 above)
Comment (mhuggins7278) [inline server.js]: I do the check in the notifyHC method but we can move it out here

File: server.js (hunk 2 above)
Comment (mhuggins7278) [inline server.js]: Yeah for some reason when I ran locally it was treating process like a special thing and erroring just changing the name seems to fix it.

File: server.js (hunk 1 above)
Comment (wballard) [inline server.js]: oh -- I get it, missed that -- makes sense

**Pull request merged**